feat(changelog): transactional-email-changed-tem 2023-01-19 (#1218)
scaleway/docs-content: changelog/january2023/2023-01-19-transactional-email-changed-tem.mdx

---
title: TEM
status: changed
author:

>     fullname: 'Join the #[transactional-email] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #transactional-email channel on Slack.'
```

>     url: 'https://scaleway-community.slack.com/archives/C0358JE0XEK'

Reviewer: ```suggestion
    url: 'https://slack.scaleway.com/'
```

date: 2023-01-19
category: managed-services
product: transactional-email
---

- Our new TEM sending Domain is now available: scw-tem.cloud: new clients will use this sending domain by default, existing clients can change to the new sending domain by modifying the SPF record on the console and then deleting the old SPF record.

> - Emails with error code 452 (i.e. recipient’s mailbox is full) will be deleted after the first attempt to send.

Reviewer 2: ```suggestion
- Emails returned with error code 452 (recipient's mailbox is full) will be deleted with no further attempts made to resend.
```

> - A minimum number of characters is now required in an email to avoid spam: the email subject and body fields must contain a minimum number of characters in order to avoid spam. The email subject must contain at least 6 characters, and the email body must contain at least 10 characters.

Reviewer 2: ```suggestion
- A minimum number of characters is now required in an email to avoid it going to spam. The subject and the body of the email must respectively contain at least 6 and 10 characters.
```
